src/main/scala/mesosphere/marathon/FeaturesConf.scala

```diff
@@ -32,13 +31,13 @@ trait FeaturesConf extends ScallopConf {
     validate = validateFeatures
   )(setStringParser)
 
-  lazy val deprecatedFeatures = opt[Set[DeprecatedFeature]](
+  lazy val deprecatedFeatures = opt[DeprecatedFeatureSet](
     "deprecated_features",
     descr = "A comma-separated list of deprecated features to continue to enable",
```

Comment:
to continue to enable -> that still should be enabled ?

Reply:
Hmm, I do feel like this wording is fine :)